### PR TITLE
make requested changes

### PR DESCRIPTION
**Pull request by jusjusjus**
@ephathaway I tried to start with the requested changes for mne-tools.

### CODE REVIEW

File: mne/io/egi/egimff.py [hunk 1]
```diff
@@ -745,14 +743,14 @@ def read_evokeds_mff(fname, condition=None, channel_naming='E%d',
 
     Notes
     -----
```
Comment (jusjusjus) [inline mne/io/egi/egimff.py:745]: I"m not sure if the Notes sectino needs to be removed now

Comment (ephathaway) [reply mne/io/egi/egimff.py:745]: From Eric's comment it looks like the `.. versionadded::` tag goes in the notes section.

**Review by ephathaway — APPROVED** [review]
These changes look good. Thanks for doing this!

**Comment by ephathaway** [comment]
Looks like the CIs are failing because we have not yet packaged `mffpy` v0.5.5.

**Pull request merged**